docker/main/rootfs/usr/local/go2rtc/create_config.py

```diff
@@ -116,6 +116,31 @@
             "-fflags nobuffer -flags low_delay -stimeout 5000000 -user_agent go2rtc/ffmpeg -rtsp_transport tcp -i {input}"
         )
 
+# add stream for each camera if user has not configured a stream already.
+# this will ensure that all features are available to the user
+if not go2rtc_config.get("streams"):
+    go2rtc_config["streams"] = {}
+
+for camera_name, camera in config.get("cameras", {}).items():
+    name = camera.get("live", {}).get("stream_name") or camera_name
+
+    # user already has the stream configured
+    if not name or name in go2rtc_config.get("streams", {}):
+        continue
+
+    inputs = camera.get("ffmpeg", {}).get("inputs")
+
+    if not inputs:
+        continue
+    elif len(inputs) == 1:
+        go2rtc_config["streams"][name] = [inputs[0]["path"]]
```

Comment:
Might be good to use the `ffmpeg:...` prefix to ensure maximum compatibility with random streams